demo/index.js

```diff
@@ -57,3 +56,61 @@ new hx.Picker('#viewMenu', {
       .selectAll('.example')
       .class('example ' + (p.value.contentClass || ''))
   })
+
+var hx = window.hx
+
+void new hx.Tabs('#tabs')
+
+function tabsTitleRenderer(node, value) {
+  hx.select(node).text('Tab ' + value)
+}
+
+function tabsContentRenderer(node, value) {
+  hx.select(node)
+    .style('padding', '1em')
+    .add(hx.detached('span').text(value.name))
+    .add(hx.label().text(value.town))
+}
+
+function setupFluidTabs() {
+  var tabItems = [
+    {
+      title: 1,
+      context: 'action',
+      content: {
+        town: 'Wheathampstead',
+        name: 'Bob'
+      }
+    }, {
+      title: 2,
+      context: 'negative',
+      content: {
+        town: 'Welwyn',
+        name: 'Kate'
+      }
+    }, {
+      title: 3,
+      context: 'positive',
+      content: {
+        town: 'Digswell',
+        name: 'Ganesh'
+      }
+    }, {
+      title: 4,
+      content: {
+        town: 'Essendon',
+        name: 'Lazlo'
+      }
+    }
+  ]
+
+  var tabs = hx.tabs({
+    items: tabItems,
+    titleRenderer: tabsTitleRenderer,
+    contentRenderer: tabsContentRenderer
+  })
+
+  hx.select('#tabs-fluid').add(tabs)
+}
+
+setupFluidTabs()
```

Comment:
This file is just for setting up things on the demo page that are not part of the examples (the dropdown that changes the background colour for example). We have put the scripts next to the corresponding piece of html to make these examples easier to read (you wouldn't structure a normal app like this, but since these are just examples, its probably fine)